app/views/admin/posts/_post.html.erb

```diff
@@ -0,0 +1,19 @@
+<tr>
+  <td><%= image_tag post.image_url.to_s %></td>
+  <td><%= post.title %></td>
+  <td>
+   <%= content_tag(:div, content_tag(:span, post_status(post.on?)[:status]), class: "status #{post_status(post.on?)[:css]}") %>
```

Comment:
sao lại dùng content_tag mà k dùng html luôn e

Reply:
dạ ở đây em xử lí thêm cả các phép toán logic mà mình thì không được code logic ở view nên em dùng cách này ạ